src/data/faq_de.json

```diff
@@ -1285,6 +1285,22 @@
+                            {
+                                "title": "TAN-Verfahren",
+                                "anchor": "tan_procedure",
+                                "active": false,
+                                "textblock": [
+                                    "<div class='container'><div class='row'><div class='col-sm'><p><b>Zweck</b></p><p>Wenn Sie auf das Sars- CoV-2-Virus getestet werden und das Ergebnis digital in der Corona-Warn-App (CWA) empfangen haben, können Sie bei einem positiven Befund in der CWA andere Nutzer*innen warnen und so helfen, ggf. Kontaktketten frühzeitig zu unterbrechen.</p><p>Wird nach dem Einscannen des QR-Codes vom Formular 10C/OEGD (oder ein äquivalentes Formular) das Ergebnis nicht in die CWA übertragen, können Sie den positiven Befund trotzdem in der App registrieren und damit andere warnen, sofern Sie eine TAN von der Verifikations-Hotline empfangen.</p></div><div class='col-sm'><a href='/assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ04_x3b.pdf' target='_blank'><img src='/assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ04_x3b.jpg' alt='Positives Testergebnis teilen Anleitung PDF'></a></div></div></div>",
+                                    "<b>Ablauf</b>",
+                                    "<ul><li>Sie erreichen die Hotline ausschließlich telefonisch.\nRufnummer: 0800-7540002 (in Deutschland) bzw. +49.30.49875402 (international)\nWann Sie die Hotline erreichen, prüfen Sie bitte hier:\n<a href='#international_phone_numbers' target='_blank'>FAQ: So erreichen Sie uns</a></li><li>Schritt 1: Beantworten Sie die Fragen des Hotline-Mitarbeiters.</li><li>Schritt 2: Sie erhalten einen Rückruf von diesem Mitarbeiter, um Ihnen die TAN mitzuteilen. Diese TAN ist eine Stunde lang gültig.</li><li>Schritt 3: Entfernen Sie, falls registriert, den Test ohne Ergebnis aus der Corona-Warn-App.</li><li>Schritt 4: Im Home Screen der CWA wählen Sie die Schaltfläche „Weiter“ in der Funktion „Sie lassen sich testen?“\nUnter der Frage „Ihr PCR-Test war positiv?“ finden Sie die Funktion „TAN für PCR-Test eingeben“, wählen Sie diese Funktion und geben Sie die 10-stellige TAN ein, die Ihnen mitgeteilt wurde.</li><li>Ihr positives Testergebnis ist nun in der App registriert. Sie können Andere warnen, indem Sie Ihre Diagnoseschlüssel teilen.</li></ul>",
+                                    "<b>Voraussetzungen und Bedingungen</b>",
+                                    "<ul><li>Um sinnvoll Begegnungen durch Contact Tracing aufzuzeichnen und diese Kontakte warnen zu können, muss die Corona-Warn-App mindestens eine Zeitlang auf dem Smartphone installiert sein; die TAN-Hotline wird diesen Zeitraum verifizieren.</li><li>Es handelt sich um einen PCR-Test, bei dem die Analyse in einem akkreditierten Labor stattfand (siehe auch Limitierungen).</li><li>Sie haben einen Laborbefund vorliegen, der die nötigen Ddetails zur Verifikation des Labors enthält.</li></ul>",

```

Comment:
Is it possible to add how long the CWA needs to be installed? I think this would be quite helpful.

Reply:
We agree that this would be helpful, but would recommend to open a seperate issue for this, as these texts were already approved and we want to go online with this today

Reply:
Agreed, I will open the issue as soon as the articles are live.